addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -350,11 +350,11 @@ private void handleRequestEvent(JSONObject data) {
             // running requests to be able to cancel it if needed
             runningRequests.put(requestId, request);
         } catch (JSONException e) {
-            logger.error(e.getMessage());
+            logger.error("{}", e.getMessage());
```

Comment:
3x I would log the exception instead, it is an error, I want to be able to debug the stacktrace